Source/ARTPresenceMap.m

```diff
@@ -119,6 +120,10 @@ - (void)internalRemove:(ARTPresenceMessage *)message {
 }
 
 - (void)internalRemove:(ARTPresenceMessage *)message force:(BOOL)force {
+    if ([message.connectionId isEqualToString:self.delegate.connectionId] && !message.isSynthesized) {
+        [_localMembers removeObject:message];
```

Comment:
Now you've added this, you need to remove line 134, or it'll do the remove unconditionally

Reply:
Fixed ca32ef2. Thanks